marathon/core/metrics.yml

```diff
@@ -19,6 +19,40 @@
                 - /core/lb
                 - /core/logging
             apps:
+            -   id: netdata
+                cmd:
+                cpus: 0.25
+                mem: 512
+                disk: 0
+                instances: "{{marathon_num_agents}}"
+                constraints:
+                - [ "hostname", "UNIQUE" ]
+                container:
+                  type: DOCKER
+                  volumes:
+                  - containerPath: "/var/run/docker.sock"
+                    hostPath: "/var/run/docker.sock"
+                    mode: RW
+                  - containerPath: "/host/proc"
+                    hostPath: "/proc"
+                    mode: RO
+                  - containerPath: "/host/sys"
+                    hostPath: "/sys"
+                    mode: RO
+                  docker:
+                    image:  "titpetric/netdata"
+                    network: BRIDGE
+                    privileged: true
+                    parameters: [{"key": "cap-add", "value": "SYS_PTRACE"}]
+                    forcePullImage: true
+                    portMappings:
+                    - containerPort: 19999
```

Comment:
I think we want to map this to a host port instead of service port. 
Do we need a registry docker, this one shall  be mapped on the nixy